packages/editor/src/pages/Editor/store/samples/sagas.ts

```diff
@@ -30,10 +29,6 @@ function* openSampleSaga(action: ActionType<typeof samples.get.request>) {
   const { content, error } = yield call(fetchYaml, url);
   if (content) {
     const solution = convertSnippetToSolution(content);
-    sendTelemetryEvent('Editor.SampleLoaded', [
-      oteljs.makeStringDataField('SampleName', solution.name),
```

Comment:
Are the sample name and ID still getting traced somewhere?

Reply:
Yup, each action can define what to send from it's payload and type, you can see the sample's data here:
https://github.com/OfficeDev/script-lab/pull/707/files#diff-04980ac347c382f169d638e6d86d67daR16